opencensus/trace/ext/httplib/trace.py

```diff
@@ -61,6 +61,13 @@ def wrap_httplib_request(request_func):
 
     def call(self, method, url, body, headers, *args, **kwargs):
         _tracer = execution_context.get_opencensus_tracer()
+        try:  # pragma: NO COVER
+            # Don't trace if it's a http request made by the exporter
+            if self._dns_host == _tracer.exporter.host_name:
```

Comment:
Probably we can make the mechanism more generic and allow httplib to accept a list of paths + hosts to not be traced. I think this is reasonable request.

@lmolkova @SergeyKanzhelev what do you think?

Reply:
Good idea. To note: the request integration has the same issues, so it would need something similar (I can implement it as well in the same PR as soon as we aggree on this part)

Reply:
Any leads on how to implement that? I don't think integrations are currently using options/parameters, so I'm not sure what would be the best way to do it.

Reply:
We have a blacklist url check method here: https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/trace/ext/utils.py#L40

And in Flask and Django integration, we allow users to provide a list of blacklisted URLs, and we will disable tracing for the blacklisted ones. Probably in here we could reuse that method.

In the exporters we actually didn't define `exporter.host_name` in the base class, could you add the param if we need it for every exporter?

Reply:
host_name is 'localhost' for several exporters by default: zipkin, jaeger, proto. Tracing calls to localhost could be legit and useful and we should prevent tracking based on something more unique like 'host:port'.

Reply:
I also wonder how this list should be shared between different  instrumentations: requests, httplib, grpc and who is responsible for configuring it? 

My assumption: each exporter should register endpoints in this list. Otherwise users are responsible for finding out all endpoints for exporters they use and registering them in all relevant instrumentations.